docs/data/date-pickers/getting-started/getting-started.md

```diff
@@ -57,70 +63,40 @@ Please note that [react](https://www.npmjs.com/package/react) and [react-dom](ht
 },
 ```
 
-### Style engine
-
-Material UI is using [Emotion](https://emotion.sh/docs/introduction) as a styling engine by default. If you want to use [`styled-components`](https://styled-components.com/) instead, run:
-
-<codeblock storageKey="package-manager">
+## Date library adapter setup
 
-```bash npm
-npm install @mui/styled-engine-sc styled-components
-```
+To integrate your chosen date library with the Date and Time Pickers, you'll need to plug the corresponding adapter into a `LocalizationProvider` that wraps your Picker components.
 
-```bash pnpm
-pnpm add @mui/styled-engine-sc styled-components
-```
+Adapters are provided for each of the supported libraries, and all are exported by both `@mui/x-date-pickers` and `@mui/x-date-pickers-pro`—for example,`AdapterDayjs`, which is used throughout this documentation for [Day.js](https://day.js.org/) integration:
```

Comment:
same as above ☝🏼

Reply:
I think that it is a separate topic from this PR that we should re-access and change the approach to if necessary. 😉

Reply:
Absolutely ... just wanted to point that out that everyone is aware of it :D